perf_timer.py

```diff
@@ -476,11 +488,11 @@ def __exit__(self, excType, excVal, excTb):
 			self.exclusive += now - self.excstart
 			self.inclusive = now - self.incstart
 
-			PerfTimer.perfQueue.append((self.scopeName, self.inclusive, self.exclusive, threading.current_thread().ident))
+			PerfTimer.perfQueue.append((self.scopeName, self.inclusive, self.exclusive, self.threadId, self.frame, self.incstart, now))
 			PerfTimer.perfStack.stack.pop()
 
 	@staticmethod
-	def PrintPerfReport(reportMode, output=None):
+	def PrintPerfReport(reportMode, output=None, name=None):
```

Comment:
Just noticed this function's docstring has not been updated with the new parameter.

Reply:
Also, in the doc for the `output` parameter, the word "tree" is missing a double quote.  And I'm not sure if the `output` doc is still correct with the addition of `name`.